src/hooks/useChainId.tsx

```diff
@@ -1,12 +1,26 @@
 // use the appropriate chain id depending on the network and if using a fork
-import { useMemo } from 'react';
+import { useContext, useMemo } from 'react';
 import { useNetwork } from 'wagmi';
+import SettingsContext from '@/contexts/SettingsContext';
 
 const DEFAULT_CHAIN_ID = 1;
 
-const useChainId = () => {
+const useChainId = (returnForkId?: boolean) => {
```

Comment:
```suggestion
const useChainId = (returnForkId = false) => {
```

Reply:
what does `returnForkId` mean?

Reply:
UseChainId will return the ID of the forked network ( eg. 1  if we forked mainnet). However, that might not be the id returned by getNetwork().id .... there will likely be a  need to have access to both. (For example, some deployed contracts might reference the id 1 , when the id returned by the network will be different ) .